### DISCUSSION TITLE
Slack channel for the app.

### DISCUSSION THREAD

**Discussion by Radi85** [Radi85/Comment#171] [Ideas]
Hey @abhiabhi94,

What do you think of creating a slack channel so that we can invite developers who would like to contribute to help them and discuss the progress?

**Comment by abhiabhi94** [comment]
As of now, we have essentially only one and a half(you one and me half) contributors to the codebase. If we have more people who want to contribute, it can be a good thing.

Anyways, if you want to start one, please go ahead. Please try to consider [zulip](https://zulip.com/) also as an alternative to slack, as it is open-source, and they claim it to be better.

**Comment by SALIMA534** [comment]
Hi , I have integrated django-comments-dab package into my project. Features of django-comments-dab package is but How to use user as our project register as not as admin user it is considering as admin user how to override this and I want to show user as my project registered users. Please help me to achieve this, Thanks.
